tests/test_bukuDb.py

```diff
@@ -69,13 +69,33 @@ def vcr_cassette_dir(request):
     return os.path.join("tests", "vcr_cassettes", request.module.__name__)
 
 
+def rmdb(*bdbs):
+    for bdb in bdbs:
+        try:
+            bdb.cur.close()
+            bdb.conn.close()
+        except Exception:
+            pass
+    if exists(TEST_TEMP_DBFILE_PATH):
+        os.remove(TEST_TEMP_DBFILE_PATH)
+
+
 @pytest.fixture()
-def setup():
+def bukuDb():
```

Comment:
Changing this fixture to a ‘managed’ `BukuDB` spawner, that ensures all connections are closed at the end of the test.